python/pfs/utils/makePfsDesign.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+from pfs.datamodel import PfsDesign, FiberStatus, TargetType
+from pfs.datamodel.utils import calculate_pfsDesignId
+
+from pfs.utils.fiberids import FiberIds
+
+
+def makePfsDesignFile(x, y,
```

Comment:
This function returns a `PfsDesign`. It doesn't write a file. So I think it should be called `makePfsDesign`.